README.md

```diff
@@ -18,39 +27,53 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+```
+node and npm
+```
+
 ### Setup
 
+```
+- clone the repo
+- cd into it
+- npm install
+```
+
 ### Install
 
+```
+open the html file
+```
+
 ### Usage
 
-### Run tests
+```
+navigate through the pages to see different contents
+```
 
 ### Deployment
 
-
+```
+set up on github pages
+```
```

Comment:
- [ ] Let's try to be accurate in our documentation as it can be of great benefit to users. In order to use the app locally users need to do one of two things:
  1. Run `npm start`

  3. Run `npm run build`. Then open the `dist/index.html` file.

  Kindly add one (or both) of these to the documentation.

Reply:
Thank you.